hazelcast/src/test/java/com/hazelcast/test/annotation/IntegrationTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+/**
+ * Tests which should be executed after a final JAR is packaged.
+ * These tests are executed by the Fail-Safe Maven plugin.
+ * (other tests are executed by the Surefire plugin)
+ *
+ */
+public final class IntegrationTest {
```

Comment:
I had spent some more time on this and I changed my mind: It's not a good idea to rely on the `Category` to distinguish between surefire and failsafe tests. 

The current categories (`QuickTest`, `SlowTest, `Nightly`) drives IF a test should be executed at all. 

However the `IntegrationTest` drives WHEN a test should be executed. These are two orthogonal concepts. 

So this is my thinking: 
1. remove the `IntegrationTest` annotation
2. make sure the current categories are respected by the failover plugin